fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerTextView.kt

```diff
@@ -297,7 +302,6 @@ internal class PeoplePickerTextView : TokenCompleteTextView<IPersona> {
         if (objects.size == personaChipLimit)
             return
 
-        shouldAnnouncePersonaAddition = true
```

Comment:
Do we need this?

Reply:
Since announcement of span removal and addition is on by default, so setting it to true is redundant.